### ISSUE TITLE
[ION-1316] Docs: local provider

### ISSUE THREAD

**Issue by jayair** [sst/sst#4236]
Document the local provider

<sub>[ION-1316](https://linear.app/sst-dev/issue/ION-1316/docs-local-provider)</sub>

**Comment by christensenjo** [comment]
What is this local provider? I'm looking around for more documentation on getting SST to talk to a local Postgres database, or maybe using the Docker provider and putting a Postgres instance in there. The only details I can really find in the docs rn are:
https://sst.dev/docs/live/#connect-to-a-local-db

I basically want to run the Drizzle / RDS guide but with a local Postgres instance instead of being billed for the RDS & its proxy.

**Comment by jayair** [comment]
Yeah this is a little different. This is basically saying there is no AWS account so store the state for my app locally.

**Comment by jayair** [comment]
Might be worth creating a separate example for your case though. If you have a snippet that works let me know and I can document that.
